CIP-0136/README.md

```diff
@@ -0,0 +1,210 @@
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/878
+Created: 2024-07-17
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Conway ledger era ushers in on-chain governance for Cardano via [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md), with the addition of many new on-chain governance artifacts.
+Some of these artifacts support the linking of off-chain metadata, as a way to provide context to on-chain actions.
+
+The [CIP-100 | Governance Metadata](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100) standard provides a base framework for how all off-chain governance metadata can be formed and handled.
+This standard was intentionally limited in scope, so that it can be expanded upon by more specific subsequent CIPs.
+
+This proposal aims to provide a specification for the off-chain metadata vocabulary that can be used to give context to Constitutional Committee (CC) votes.
+
+## Motivation: why is this CIP necessary?
+
+The high-level motivation for this proposal is to provide a standard which improves legitimacy of Cardano's governance system.
+
+### Clarity for governance action authors
+
+Governance action authors are likely to have dedicated a significant amount of time to making their action meaningful and effective (as well as locking a significant deposit).
+If this action is not able to be ratified by the CC, it is fair for the author to expect a reasonable explanation from the CC.
+
+Without reasonable context being provided by the CC votes, authors may struggle to iterate upon their actions, until they are deemed constitutional.
+This situation could decrease perceived legitimacy in Cardano's governance.
+
+### Context for other voting bodies
+
+By producing a standard we hope to encourage all CC members to attach rich contextual metadata to their votes.
+This context should show CC member's decision making is fair and reasonable.
+
+This context allows the other voting bodies to adequately check the power of the CC.
+
+### CC votes are different to other types of vote
+
+The CC and their votes are fundamentally very different from the other voting bodies.
+This makes reusing standards from these voting bodies problematic.
+
+### Inclusion within interim constitution
+
+Cardano's [Interim Constitution Article VI Section 4](https://github.com/IntersectMBO/interim-constitution/blob/75155526ce850118898bd5eacf460f5d68ceb083/cardano-constitution-0.txt#L330) states:
+
+```txt
+Constitutional Committee processes shall be transparent.
+The Constitutional Committee shall publish each decision.
+When voting no on a proposal, the Committee shall set forth the basis
+for its decision with reference to specific Articles of this Constitution
+that are in conflict with a given proposal.
+```
+
+This mandates that the CC must provide rationale for at least `no` votes.
```

Comment:
```suggestion
This mandates that the CC must provide rationale for `Yes, `No` and `Abstain` votes.
```
Some might not realize the true meaning behind the machine-readable "no" in this context due to the way it is written, preventing any confusion or unnecessary effort to explain their decision when abstaining from voting.

Kudos to the authors of this CIP! I truly appreciate the meticulous effort they put into it. 😄👍